packages/design/src/Popover/Popover.jsx

```diff
@@ -129,11 +129,27 @@ export default class Popover extends React.Component {
 
   setPositioningStyles = element => {
     const positioning = this.getPositioningStyle(element);
-    if (positioning.top !== null) {
-      element.style.top = positioning.top;
+    // we don't want so set both top/bottom values (it causes the scrollbar to show when height increases)
+    // if the Popover is aligned to bottom, then set only `bottom` value
+    // if the Popover is aligned to center or top, then set only `top` value
+    // the same applies to left/right alignment
+    if (this.props.transformOrigin.vertical === 'bottom') {
+      if (positioning.bottom) {
+        element.style.bottom = positioning.bottom;
+      }
+    } else {
+      if (positioning.top) {
+        element.style.top = positioning.top;
+      }
     }
-    if (positioning.left !== null) {
-      element.style.left = positioning.left;
+    if (this.props.transformOrigin.horizontal === 'right') {
+      if (positioning.right) {
+        element.style.right = positioning.right;
+      }
+    } else {
+      if (positioning.left) {
+        element.style.left = positioning.left;
+      }
     }
```

Comment:
What's the reason behind tying top-left/bottom-right to `props.transformOrigin`?

`transformOrigin` dictates the point of the popover that will attach to the anchor's origin. Using top-left vs bottom-right dictates how the component will grow in height and width. I can imagine situations where someone would want those two to not be coupled to each other.

Tying this to `props.transformOrigin` also means that we're changing how the Popover behaves not only in the case of the feedback form but also every other instance of popover that used right or bottom for its `transformOrigin`.

Perhaps a safer and more extensible approach would be to introduce a new prop? For now it wouldn't have to be super-extensible and handle all imaginable cases. Something like `positionalProperties` which accepts either `top-left` (default) or `bottom-right`.

Reply:
In case of adding new props, please remember to add new items to the story as well.

Reply:
Ah, I was thinking about it incorrectly. You're absolutely right, `top-left`/`bottom-right` shouldn't be tied to `props.transformOrigin`. 
I added `growDirections` prop to the popover with the values you proposed (`bottom-right` is a default value).
There is also an updated story for this.